docs/CloudPermissions.md

```diff
@@ -0,0 +1,56 @@
+# Restricting cloud privileges for deploying Scalar DL
+
+This guide explains how to restrict privileges for deploying Scalar DL on Azure.
+
+In general from security perspective, it is better to assign restricted privileges to users not to make them able to do operations that they are not supposed to do.
+
+## Azure
+
+The following JSON is a custom role that allow users to manage resources for Scalar DL. This is sufficient but not necessary.
+
+In Azure Portal, you can create the role in Subscriptions section. Choose your subscription and select Access control (IAM) from the menu, then click `+Add` and select `Add custom role`. 
+Once the role is created, you can assign it to a user or a group from `Add role assignment` in the `+Add` menu.
+
+Please keep your subscription ID in the `assignableScopes` array. You can give `rolename` and `description` as per your convenience.
+
+```json
+{   
+ "properties": {
+        "roleName": "manual-guide-test",
+        "description": "manual-guide-test",
+        "assignableScopes": [
+            "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+        ],
+        "permissions": [
+            {
+                "actions": [
+                    "Microsoft.Authorization/roleAssignments/*",
+                    "Microsoft.Compute/availabilitySets/*",
+                    "Microsoft.Compute/disks/*",
+                    "Microsoft.Compute/virtualMachines/*",
+                    "Microsoft.Network/loadBalancers/*",
+                    "Microsoft.Network/networkInterfaces/*",
+                    "Microsoft.Network/networkSecurityGroups/*",
+                    "Microsoft.Network/publicIPAddresses/*",
+                    "Microsoft.Network/virtualNetworks/*",
+                    "Microsoft.Resources/deployments/*",
+                    "Microsoft.Resources/subscriptions/resourceGroups/*",
+                    "Microsoft.DocumentDb/databaseAccounts/*",
+                    "Microsoft.ContainerService/managedClusters/*",
+                    "Microsoft.OperationsManagement/solutions/*",
+                    "Microsoft.operationalInsights/workspaces/*"
+                ],
+                "notActions": [],
+                "dataActions": [],
+                "notDataActions": []
+            }
+        ]
+    }
+}
+```
+
+### A Role for creating AKS cluster
```

Comment:
This section talks about a role? 
It doesn't look like it. Please fix it.

Reply:
Changed heading to `Create Privileges for AKS cluster` in c601da6